grpc-server-micrometer/src/main/scala/com/avast/sst/grpc/server/micrometer/MonitoringServerInterceptor.scala

```diff
@@ -0,0 +1,59 @@
+package com.avast.sst.grpc.server.micrometer
+
+import java.util.concurrent.atomic.AtomicLong
+import java.util.concurrent.{ConcurrentHashMap, TimeUnit}
+
+import io.grpc.ForwardingServerCall.SimpleForwardingServerCall
+import io.grpc._
+import io.micrometer.core.instrument.{MeterRegistry, Timer}
+
+/** Records important gRPC call metrics in [[io.micrometer.core.instrument.MeterRegistry]].
+  *
+  * Metrics:
+  *   - grpc.<method>.current-calls
```

Comment:
IMHO we must provide _absolute_ metric name in Micrometer (it is not possible to _nest_ metrics names). So I would prefer to have `grpc.<service>.<method>.*`

Reply:
According to the docs `getFullMethodName` returns fully qualified name including the service name so we should be fine.